.idea/inspectionProfiles/Project_Default.xml

```diff
@@ -0,0 +1,6 @@
+<component name="InspectionProjectProfileManager">
```

Comment:
I don't think we should be checking `.idea` files into the project... I think this should be in `.gitignore`?